### PR TITLE
Release Allo V2 🚀 

### PR DESCRIPTION
**Pull request by boudra**
fixes https://github.com/gitcoinco/grants-stack/issues/3126
fixes https://github.com/gitcoinco/grants-stack/issues/3125
fixes https://github.com/gitcoinco/grants-stack/issues/3124
fixes https://github.com/gitcoinco/grants-stack/issues/3123
fixes https://github.com/gitcoinco/grants-stack/issues/3127

before merging:
- [ ] change default ALLO_VERSION env var to v2 in Vercel
- [x] set up explorer-v1.gitcoin.co

### CODE REVIEW

**Review by thelostone-mc — APPROVED** [review]

**Review by bhargavaparoksham — COMMENTED** [review]
codewise looks good. 

@boudra fyi In the UI i'm able to toggle between v1 & v2 in staging but i'm not able to toggle in prod.

**Review by melissa-neira — APPROVED** [review]
looks good to me!

**Review by michellema1208 — CHANGES_REQUESTED** [review]
hey @boudra - sharing some quick feedback testing the switch between v1/v2: 

**Functionality**
- [ ] On both Builder and Manager, it seems like the switch to v1 is not working -- when I select v1, the page reloads but then I'm still connected to v2. Similarly if I'm on Manager on v2 > click on v1 round --> I'm stuck in an infinite loop of switching to v1. 

**UI** + @melissa-neira for feedback as well 

- [ ] I'm noticing that the white dropdown blends in with the Builder background given Builder is default white background. Should we add an outline or the grey shading that we have for the switch dropdown on Manager?

Screen Shot 2024-03-20 at 1 13 45 PM

- [ ] Speaking of Manager, it seems like the dropdown UI across Builder and Manager is inconsistent - which fonts/patterns should we stick too? 

Builder: 
Screen Shot 2024-03-20 at 1 13 45 PM

Manager: 
Screen Shot 2024-03-20 at 1 20 22 PM

- [ ] Explorer Donation History page is showing different stats on v2 versus v1 -- my understanding is that on both sites you should see all of your donations across v1 and v2?

I'm seeing different stats when I load the same contributor profile on: 

- https://explorer-git-allo-migration-grants-stack.vercel.app/#/contributors/0x37e5e6eb36467F98392ebBb35F110FbB8238e6D3
- https://explorer-v1.gitcoin.co/#/contributors/0x37e5e6eb36467F98392ebBb35F110FbB8238e6D3

File: packages/grant-explorer/src/features/api/AlloWrapper.tsx [hunk 1]
```diff
@@ -43,7 +42,6 @@ function AlloWrapper({ children }: { children: JSX.Element | JSX.Element[] }) {
           waitUntilIndexerSynced: createWaitForIndexerSyncTo(
             `${getConfig().dataLayer.gsIndexerEndpoint}/graphql`
           ),
-          allo: zeroAddress,
```
Comment (thelostone-mc) [inline packages/grant-explorer/src/features/api/AlloWrapper.tsx:46]: ^ this seem to be breaking builder

**Comment by boudra** [comment]
@michellema1208 

- the switcher is now fixed
- i made the dropdown slightly grey-ish on builder (@melissa-neira please let me know if that's fine or you want to do something else!)
- fonts should now be consistent across app to use DM Mono
- stats should now be the same, the difference was just because v1 was also loading testnets, but they are both showing v1 and v2

**Review by michellema1208 — APPROVED** [review]

**Pull request merged**